ui-frontend/packages/catalog-ui-search/src/main/webapp/component/fields/near.tsx

```diff
@@ -23,6 +24,36 @@ type NearFieldProps = {
   onChange: (val: ValueTypes['proximity']) => void
 }
 
+const showCustomTextFieldOrDefault = ({
```

Comment:
Is it possible to factor this out into a common component between near and the filterInput?

Reply:
Yeah, I think so since they seem to be more or less the same thing 👍